release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2852,3 +2852,27 @@ link:https://access.redhat.com/solutions/6327721[{product-title} 4.7.30 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-31"]
+=== RHBA-2021:3510 - {product-title} 4.7.31 bug fix update
+
+Issued: 2021-09-21
+
+{product-title} release 4.7.31 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3510[RHBA-2021:3510] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:3509[RHBA-2021:3509] advisory.
```

Comment:
I'm connected to VPN, but getting 404 error pages for these links. Is that expected?

Reply:
Yes, they should work after the release on Tuesday.